### ISSUE TITLE
Under what circumstances will Gatsby automatically add a browser prefix for css?

### ISSUE THREAD

**Issue by LuckyRabbitFeet** [gatsbyjs/gatsby#20467]
## Summary

For example `display: flex;`。
I try to add `" browserslist ": ["> 0.25% "," not dead "]` to package.json, the output of `display: flex;` is not prefixed with a browser. But if I use gatsby- plugin-postcss and autoprefixer, `display: -webkit-box;` will be added to the output. 

Am I missing any configuration?

### Environment 

System: windows10 1809

Node: v11.13.0
npm: 6.9.0

Browsers:
Chrome: 79.0.3945.88
Firefox: 73.0b1

Gatsby CLI version: 2.8.23
Gatsby version: 2.18.18

### File contents

`gatsby-config.js`: 
`
module.exports = {
  ...
  plugins: [
    // 'gatsby-plugin-postcss',
    ...
  ],
}
`

`package.json`: 
`
{
  ...
  "browserslist": [">0.25%", "not dead"]
}
`

`postcss.config.js`
`
module.exports = () => ({
  plugins: {
    'autoprefixer': {},
  },
})
`

**Comment by freiksenet** [comment]
Hello!

Thank you for your issue! I wonder if you could create a [minimal reproduction](https://www.gatsbyjs.org/contributing/how-to-make-a-reproducible-test-case/) where this bug exists? 

Thank you! :purple_heart:

**Comment by LuckyRabbitFeet** [comment]
> Hello!
> 
> Thank you for your issue! I wonder if you could create a [minimal reproduction](https://www.gatsbyjs.org/contributing/how-to-make-a-reproducible-test-case/) where this bug exists?
> 
> Thank you! 💜

Thank you for your reply.
After I created the minimal reproduction, I reorganized the problem, and I found that the automatic prefix function is normal, but the only `display: flex;` is not the same. I tested it through the https://autoprefixer.github.io/ website, under the condition of `"browserslist": [">0.25%", "not dead"]` , `Display: flex;` should be able to output `display: -webkit-box;`, but not. 
Is this normal and new?

I have created a minimal copy, the address is https://github.com/LuckyRabbitFeet/gatsby-bug-repro.git
